mod_reforged/hooks/states/tactical_state.nut

```diff
@@ -1,4 +1,30 @@
 ::Reforged.HooksMod.hook("scripts/states/tactical_state", function(q) {
+	q.initMap = @(__original) function()
+	{
+		// Feat: the same location will always produce the exact same layout
+		// This has no influence over troop spawning as those will be seeded  by "combatSeed" later on
+		if (this.m.StrategicProperties.MapSeed == 0)	// Other fixed seeds take priority
+		{
+			foreach (party in this.m.StrategicProperties.Parties)
+			{
+				if (party.isLocation())
+				{
+					::Reforged.Math.seedRandom(
+						"RF_FixedLocationSeed",		// Fixed salt, specific to use-case
+						party.getTypeID(),			// Location specific salt
+						party.getName(),			// Location specific salt
+						party.getFaction(),			// Faction specific salt
+						party.getTile().X * 200,	// Position specific salt
+						party.getTile().Y			// Position specific salt
+					);
+				}
+				break;
```

Comment:
The `break;` was added in the wrong place, it needs to be inside the `if` block.